cloudburst/server/scheduler/call.py

```diff
@@ -45,7 +45,9 @@ def call_function(func_call_socket, pusher_cache, policy):
     refs = list(filter(lambda arg: type(arg) == CloudburstReference,
                        map(lambda arg: serializer.load(arg),
                            call.arguments.values)))
-    result = policy.pick_executor(refs)
+    result = None
```

Comment:
This change might have some unintended consequences: What this will basically do is try to pick executors that we believe are already overloaded over and over again. It will not allow new executors back into the system because you're going to loop until you pick an executor.

Reply:
This is also why I suggested having a separate policy for local mode.